using-disks.tex

```diff
@@ -343,7 +343,7 @@ \subsection{DELETE}
 
 \subsection{Shortcut Disk Commands}
 
-BASIC 65 provides several shortcuts for common disk commands for use from the \screentext{READY} prompt.
+BASIC 65 provides several shortcuts for common disk commands for use from the \screentext{READY} prompt. Remember though, shortcuts are run on the currently set default unit.
```

Comment:
I don't think this fact is represented elsewhere, so this can be a factual statement, not a reminder: "Shortcuts always run on the currently set default unit. See \textbf{SET DEF}." (And maybe this is a separate short paragraph in this section, and maybe "see" is a linked reference to the appendix entry.)

Reply:
Agree!